client/layout/index.jsx

```diff
@@ -34,7 +34,7 @@ import QueryPreferences from 'components/data/query-preferences';
 import PropTypes from 'prop-types';
 import QuerySites from 'components/data/query-sites';
 import { isOffline } from 'state/application/selectors';
-import { hasSidebar, masterbarIsVisible } from 'state/ui/selectors';
```

Comment:
Instead of removing this, we could modify `hasSidebar()` to check for the existence of `context.secondary` and not the _section configs_ `secondary` that we're going to remove from everywhere. See https://github.com/Automattic/wp-calypso/pull/19494 for details about `context.secondary`.

https://github.com/Automattic/wp-calypso/blob/0c6e9e281edd578b23ab4a33d0a0c570950f0f38/client/state/ui/selectors.js#L129-L136

Reply:
Note that in this file we want to set `has-no-sidebar` class and by simply removing `hasSidebar` selector things would break: https://github.com/tam-nguyen/wp-calypso/blob/024341fbd6a4e1ca2f737e0c6ccb381a865276ff/client/layout/index.jsx#L99-L109